cfme/containers/provider/__init__.py

```diff
@@ -310,14 +346,23 @@ def resetter(self):
 
 @navigator.register(ContainersProvider, 'Edit')
 class Edit(CFMENavigateStep):
-    VIEW = ContainersProviderEditView
+
+    def pickcontainerprovidereditview(self):
```

Comment:
use `_` here to make this a bit more readable, consider `container_provider_edit_view_class`.

Calling it `pick(...)` is misleading since it doesn't actually pick, it returns a VersionPick object. The method could take a version argument and call pick before returning, where the caller passes appliance.version

Reply:
:+1: